src/tests/unit/tests/DetailsView/components/save-assessment-button.test.tsx

```diff
@@ -36,71 +44,95 @@ describe('SaveAssessmentButton', () => {
             href: 'url',
             userConfigurationStoreData,
         };
-        eventStub = new EventStubFactory().createMouseClickEvent();
     });
 
     describe('on dialog enabled', () => {
-        let wrapper: ShallowWrapper;
+        describe('render', () => {
+            beforeEach(() => {
+                wrapper = render(<SaveAssessmentButton {...propsStub} />);
+                fireEvent.click(wrapper.getByRole('link'));
```

Comment:
We can still click on the mocked element and trigger the onclick.
```suggestion
                fireEvent.click(wrapper.container.querySelector('mock-customizedactionbutton'));
```